updating description in DLS and FLS docs (#11594)
opensearch-project/documentation-website: _security/access-control/document-level-security.md

---

# Document-level security

> Document-level security (DLS) determines the documents that a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents on an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.

Reviewer: ```suggestion
Document-level security (DLS) determines the documents that a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents in an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.
```

---

# Document-level security

> Document-level security (DLS) controls which documents a role can retrieve when reading from an index. It applies only to read operations, such as search and get, and does not restrict indexing, updating, or deleting documents. If a role has write or delete permissions on an index, it can still modify or remove documents that are hidden by DLS. The write permissions are governed solely by index permissions and action groups. To get started with document level security, open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index permissions** section, shown in the following image.

Reviewer 2: ```suggestion
Document-level security (DLS) controls which documents a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents on an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.

To get started with DLS, open OpenSearch Dashboards and choose **Security**. Then select **Roles**, create a new role, and review the **Index permissions** section shown in the following image.
```

## Simple roles

> Document-level security uses OpenSearch query domain-specific language (DSL) to define which documents a role can retrieve. In OpenSearch Dashboards, choose an index pattern and provide a query in the **Document-level security** section:

Reviewer 2: ```suggestion
Document-level security (DLS) uses OpenSearch query DSL to define the documents that a role is allowed to retrieve. In OpenSearch Dashboards, choose an index pattern and provide a query in the **Document-level security** section:
```
